example/provider-local/base/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+secretGenerator:
+- name: seed-local
+  namespace: garden
+  files:
+  - kubeconfig
+
+resources:
+- cloudprofile.yaml
+- controller-registration.yaml
+- project.yaml
+- secretbinding.yaml
+
+patchesStrategicMerge:
+- patch-controller-registration.yaml
```

Comment:
The `controller-registration.yaml` is generated via `//go:generate`, hence, we cannot modify it directly to add the `Always` deployment policy. An alternative would have been to extend the generation script for such purpose, however, this would make it even more complex given that there are many more configuration options one might want to specify in a `ControllerRegistration`.